packages/compass-connections/src/index.tsx

```diff
@@ -34,6 +33,7 @@ const ConnectionsComponent: React.FunctionComponent<{
   onAutoconnectInfoRequest?: (
     connectionStorage: ConnectionStorage
   ) => Promise<ConnectionInfo | undefined>;
+  allowAutoconnectInfoReconnect?: boolean;
```

Comment:
nit: Maybe it's me, it's still a bit hard to understand what the purpose of this boolean is from the name. Is there another name we could give it? 

Thinking of a name from my understanding, it ends up being super long, so maybe this is fine as is. 
`allowReconnectWhenConnectionInfoHasAutoconnected`?

Reply:
Yeah, fair, I'll update, your proposal is closer, but still not there a bit (I'm thinking if I flip the value it maybe will be better? `doNotReconnectDisconnectedAutoconnectInfo`, that's like exactly what we're trying to do here I guess even if it's super verbose), it is getting long, but it's a corner case handling thing so probably okay 🙂

Reply:
Inverting it makes sense to me 👍